app/scripts/contentscript.js

```diff
@@ -20,8 +20,14 @@ const CONTENT_SCRIPT = 'metamask-contentscript'
 const INPAGE = 'metamask-inpage'
 const PROVIDER = 'metamask-provider'
 
+// TODO:LegacyProvider: Delete
+const LEGACY_CONTENT_SCRIPT = 'contentscript'
+const LEGACY_INPAGE = 'inpage'
+const LEGACY_PROVIDER = 'provider'
+const LEGACY_PUBLIC_CONFIG ='publicConfig'
+
 if (shouldInjectProvider()) {
-  injectScript(inpageBundle)
+  // injectScript(inpageBundle)
```

Comment:
Leftover from debugging
```suggestion
  injectScript(inpageBundle)
```